### DISCUSSION TITLE
Более 5 аккаунтов

### DISCUSSION THREAD

**Discussion by pashva11** [masterking32/MasterHamsterKombatBot#83] [Q&A] [answered]
"auto_playground_games": True,  # Enable auto playground games by setting it to True, or set it to False to disable
            # If you have more than 5 accounts, you need to disable the auto_playground_games feature or use a proxy for each account.

А что будет, если у меня больше 5 аккаунтов и они все будут работать без прокси?

**Comment by masterking32** [comment, marked as answer]
> "auto_playground_games": True, # Enable auto playground games by setting it to True, or set it to False to disable # If you have more than 5 accounts, you need to disable the auto_playground_games feature or use a proxy for each account.
> 
> А что будет, если у меня больше 5 аккаунтов и они все будут работать без прокси?

Maybe the application gets stuck. Don't enable that feature for more than five accounts. Just enable it for the first five accounts. You can create only 25 tokens per day, not more.
